tools/db_crashtest.py

```diff
@@ -73,6 +73,8 @@
     "expected_values_dir": lambda: setup_expected_values_dir(),
     "fail_if_options_file_error": lambda: random.randint(0, 1),
     "flush_one_in": 1000000,
+    "manual_wal_flush": lambda: random.randint(0, 1),
+    "manual_wal_flush_one_in": 1000000,
```

Comment:
One per million seems too infrequent to expose race conditions. How about `lambda: random.choice([0, 0, 1000, 1000000])`?